tenant-manager-client/src/main/java/org/bf2/srs/fleetmanager/spi/tenants/impl/RestClientTenantManagerServiceImpl.java

```diff
@@ -188,7 +190,10 @@ public Optional<Tenant> getTenantById(TenantManagerConfig tm, String tenantId) t
     public List<Tenant> getAllTenants(TenantManagerConfig tm) throws TenantManagerServiceException {
         try {
             var client = getClient(tm);
-            return client.listTenants().stream()
```

Comment:
The old code return all tenants, the new code does pagination that needs to be handled

Reply:
Otherwise the whole method will have to be paginated

Reply:
is this enough to target your comment @jsenko ?
https://github.com/bf2fc6cc711aee1a0c2a/srs-fleet-manager/pull/313/commits/b4975181450bd806f897355409226d5b5dc04aa2